docs/integrations/github/org.md

```diff
@@ -168,7 +168,7 @@ const backend = createBackend();
 // Other items
 
 backend.add(import('@backstage/plugin-catalog-backend/alpha'));
-
+backend.add(import('@backstage/plugin-catalog-backend-module-github-org'));
```

Comment:
What's the difference between this line and the one after?

I think we should highlight the difference in the docs, otherwise it could be confusing for people. :pray: